Spoon Master/Scenes/Main/CollectionViewCell/HomeCollectionViewCell.swift

```diff
@@ -7,12 +7,41 @@
 //
 
 import UIKit
+import Kingfisher
+import Then
 
-final class HomeCollectionViewCell: UICollectionViewCell {
+enum DataCell {
+    case recipesCell(Recipe)
+    case productsCell(Item)
+}
 
+final class HomeCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet private weak var avatarOfFood: UIImageView!
+    @IBOutlet private weak var nameOfFood: UILabel!
+        
     override func awakeFromNib() {
         super.awakeFromNib()
-        // Initialization code
+        avatarOfFood.do {
```

Comment:
```suggestion
        resultImageView.layer.do {
```
Cũng tạo 1 hàm configView